meta-luneos/conf/distro/include/luneos.inc

```diff
@@ -122,7 +122,10 @@ OVERRIDES[vardepsexclude] += "WEBOS_EXTRA_MACHINEOVERRIDES"
 
 INHERIT_DISTRO ?= "debian devshell sstate license remove-libtool buildstats buildstats-summary own-mirrors image-buildinfo"
 
-SOURCE_MIRROR_URL = "http://sources.webos-ports.org/"
+# Set to "" because build server not being available and leading to errors on clean builds.
+# SOURCE_MIRROR_URL = "http://sources.webos-ports.org/"
+SOURCE_MIRROR_URL = ""
```

Comment:
Setting it to empty will add bogus entries to PREMIRROR with https://git.openembedded.org/openembedded-core/tree/meta/classes/own-mirrors.bbclass

Better to keep it set and just remove own-mirrors from INHERIT_DISTRO.